plugins/vacation-calendar/src/components/CalendarCard/CalendarCard.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { ReactNode, useState } from 'react';
+import useAsync from 'react-use/lib/useAsync';
+import Timeline, {
+  TimelineHeaders,
+  SidebarHeader,
+  DateHeader,
+  IntervalRenderer,
+} from 'react-calendar-timeline';
+import { DateTime } from 'luxon';
+import { useApi } from '@backstage/core-plugin-api';
+import { useEntity, catalogApiRef } from '@backstage/plugin-catalog-react';
+import {
+  Content,
+  ContentHeader,
+  SupportButton,
+  Link,
+  ErrorPanel,
+  Progress,
+  Avatar,
+} from '@backstage/core-components';
+import { DateSelector } from '../DateSelector';
+import { getGroups, getScheduleItems } from './lib';
+import { fetchGroupEntities, fetchUserEntities } from './fetch';
+import { useAvailability } from '../../hooks/useAvailibility';
+import { useSignIn } from '../../hooks';
+import { SignInContent } from '../SignInContent';
+import MuiLink from '@mui/material/Link';
+import Button from '@mui/material/Button';
+import Box from '@mui/material/Box';
+import Typography from '@mui/material/Typography';
+import Stack from '@mui/material/Stack';
+import { makeStyles } from '@material-ui/core/styles';
+import { Theme } from '@mui/material/styles';
+
+const DEFAULT_NUM_DAYS = 60;
+
+const useStyles = makeStyles({
```

Comment:
This is the only usage of "@material-ui/core"? Let's try to remove this deprecated dependency?

Reply:
Backstage Avatar component documentation uses material-ui/core, so that is why I used it. 

But I replaced it now with the customStyles prop. This prop is deprecated, but we use it everywhere else so let's go with it.